nncf/experimental/post_training/api/metric.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
This is not related to this PR specifically but I noticed that NNCF copyrights format is different from what OpenVINO uses now.

Reply:
Should we schedule a global copyright update? Create a ticket?